website.Rmd

```diff
@@ -2,17 +2,189 @@
+
+Let's walk through what `use_pkgdown_github_pages()` actually does:
+
+-   Initializes an empty, "orphan" branch in your GitHub repo, named `gh-pages` (for "GitHub Pages").
+    The `gh-pages` branch will only live on GitHub (there's no reason to fetch it to your local computer) and it represents a separate, parallel universe from your actual package source.
+    The only files tracked in `gh-pages` are those that constitute your package's website (the files that you see locally below `docs/`).
+
+-   Turns on GitHub Pages for your repo and tells it to serve a website from the files found in the `gh-pages` .
+
+-   Copies the configuration file for a GHA workflow that does pkgdown "build and deploy".
+    The file shows up in your package as `.github/workflows/pkgdown.yaml`.
+    If necessary, some related additions are made to `.gitignore` and `.Rbuildignore`.
+
+-   Adds the URL for your site as the homepage for your GitHub repo.
+
+-   Adds the URL for your site to `DESCRIPTION` and `_pkgdown.yml`.
+    The autolinking behaviour we've touted elsewhere relies on your package listing its URL in these two places, so this is a high-value piece of configuration.
+
+After successful execution of `use_pkgdown_github_pages()`, you should be able to visit your new site at the URL displayed in the output above.[^website-1]
+By default the URL has this general form: `https://USERNAME.github.io/REPONAME/`.
+
+[^website-1]: Sometimes there's a small delay, so give it up to a couple of minutes to deploy.
+
+## Now what?
+
+For a typical package, you could stop here --- after creating a basic pkgdown site and arranging for it to be re-built and deployed regularly --- and people using (or considering using) your package would benefit greatly.
+Everything beyond this point is a "nice to have".
+
+Overall, we recommend `vignette("pkgdown", package = "pkgdown")` as a good place to start, if you think you want to go beyond the basic defaults.
+
+In the sections below, we highlight a few areas that are connected to other topics in the book or customizations that are particularly rewarding.
+
 ## Logo
 
-Website is also a great way to show off your package logo.
+It's fun to have a package logo!
+In the R community, we have a strong tradition of hex stickers, so it can be nice to join in with a hex logo of your own.
+Keen R user Amelia McNamara [made herself a dress](https://www.amelia.mn/blog/misc/2019/08/17/Tidy-Dress.html) out of custom hex logo fabric and useR!
+2018 featured a [spectacular hex photo wall](https://www.mitchelloharawild.com/blog/hexwall/).
+
+Here are some resources to guide your logo efforts:
+
+-   The convention is to orient the logo with a vertex at the top and bottom, with flat vertical sides.
+
+-   If you think you might print stickers, make sure to comply with the de facto standard for sticker size.
```

Comment:
I think you should inline the size advice here (at least the aspect ratio)

Reply:
added the official "spec" as an image